tests/utils/asserts.ts

```diff
@@ -101,6 +101,13 @@ export async function assertImageSize(
   expect(actual?.height).toEqual(height);
 }
 
+export async function assertImageOption(page: Page) {
+  const actual = await page
+    .locator('.affine-embed-editing-state-container')
+    .count();
```

Comment:
Maybe should use [await expect(locator).toBeVisible()](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible)